config/routes.rb

```diff
@@ -3,7 +3,11 @@
 
   namespace :api do
     namespace :v1 do
-      resources :users, only: [:show]
+      resources :users, only: [:show, :update, :create] do
+        resources :events, only: [:index], controller: 'users/events'
```

Comment:
Good use of `controller:` to specify which controller. Good use of specifying `'users/events'` to use a namespaced controller.